### PR TITLE
Updated The pma_scan_list for more accuracy

### PR DESCRIPTION
**Pull request by aman566**
Updated the list of pma_scan_list

#### Checklist
- [X] I have followed the [Contributor Guidelines](https://github.com/zdresearch/OWASP-Nettacker/wiki/Developers#contribution-guidelines).
- [X] I have added the relevant documentation.
- [X] My branch is up-to-date with the Upstream master branch.

#### Changes proposed in this pull request
- [X] Added more words to pma_scan_list
#### Your development environment
- OS: `Linux`
- OS Version: `Ubuntu`
- Python Version: `3.6.9`

### CODE REVIEW

File: lib/scan/pma/engine.py [hunk 1]
```diff
@@ -18,34 +18,46 @@ def extra_requirements_dict():
...
-                          '/PMA/', '/pma/', '/dbadmin/', '/mysql/', '/myadmin/', '/phpmyadmin2/', '/phpMyAdmin2/',
-                          '/phpMyAdmin-2/', '/php-my-admin/', '/phpMyAdmin-2.2.3/', '/phpMyAdmin-2.2.6/',
-                          '/phpMyAdmin-2.5.1/', '/phpMyAdmin-2.5.4/', '/phpMyAdmin-2.5.5-rc1/',
-                          '/phpMyAdmin-2.5.5-rc2/', '/phpMyAdmin-2.5.5/', '/phpMyAdmin-2.5.5-pl1/',
-                          '/phpMyAdmin-2.5.6-rc1/', '/phpMyAdmin-2.5.6-rc2/', '/phpMyAdmin-2.5.6/',
-                          '/phpMyAdmin-2.5.7/', '/phpMyAdmin-2.5.7-pl1/', '/phpMyAdmin-2.6.0-alpha/',
-                          '/phpMyAdmin-2.6.0-alpha2/', '/phpMyAdmin-2.6.0-beta1/', '/phpMyAdmin-2.6.0-beta2/',
-                          '/phpMyAdmin-2.6.0-rc1/', '/phpMyAdmin-2.6.0-rc2/', '/phpMyAdmin-2.6.0-rc3/',
-                          '/phpMyAdmin-2.6.0/', '/phpMyAdmin-2.6.0-pl1/', '/phpMyAdmin-2.6.0-pl2/',
+        "pma_scan_list": ['/admin/', '/accounts/login/', '/admin1.php/', '/admin.php/',
```
Comment (Ali-Razmjoo) [inline lib/scan/pma/engine.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/zdresearch/OWASP-Nettacker/pullRequest?prid=5065955)

**Comment by Ali-Razmjoo** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/zdresearch/OWASP-Nettacker/pullRequest?prid=5065955&bid=16668947)

File: lib/scan/pma/engine.py [hunk 2]
```diff
@@ -18,34 +18,46 @@ def extra_requirements_dict():
...
+                          '/sql/sqlweb/', '/sql/websql/', '/sql/webdb/', '/sql/sqladmin/', '/sql/sql-admin/',
+                          '/sql/phpmyadmin2/', '/sql/phpMyAdmin2/', '/sql/phpMyAdmin/', '/db/myadmin/', '/db/webadmin/',
+                          '/db/websql/', '/db/dbadmin/', '/db/db-admin/', '/db/phpmyadmin3/', '/db/phpMyAdmin3/',
+                          '/db/phpMyAdmin-3/', '/administrator/phpmyadmin/', '/administrator/phpMyAdmin/',
+                          '/administrator/pma/', '/administrator/PMA/', '/phpMyAdmin3/', '/phpMyAdmin4/', '/phpMyAdmin-3/',
+                          '/PMA2011/', '/PMA2012/', '/PMA2013/', '/PMA2014/', '/PMA2015/', '/PMA2016/', '/PMA2017/',
+                          '/PMA2018/', '/pma2011/', '/pma2012/', '/pma2013/', '/pma2014/', '/pma2015/', '/pma2016/',
+                          '/pma2017/', '/pma2018/', '/phpmyadmin2011/', '/phpmyadmin2012/', '/phpmyadmin2013/',
+                          '/phpmyadmin2014/', '/phpmyadmin2015/', '/phpmyadmin2016/', '/phpmyadmin2017/', '/phpmyadmin2018/',]
+
```
Comment (Ali-Razmjoo) [inline lib/scan/pma/engine.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/zdresearch/OWASP-Nettacker/pullRequest?prid=5065955)

Comment (aman566) [reply lib/scan/pma/engine.py]: Fixed the whitespace mistake

**Comment by Ali-Razmjoo** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/zdresearch/OWASP-Nettacker/pullRequest?prid=5065955&bid=16668947)

File: lib/scan/pma/engine.py [hunk 3]
```diff
@@ -19,33 +19,45 @@ def extra_requirements_dict():
...
```
Comment (Ali-Razmjoo) [inline lib/scan/pma/engine.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/zdresearch/OWASP-Nettacker/pullRequest?prid=5065955)

**Comment by aman566** [comment]
Implemented in #272 closing for better commit history.

**Pull request closed without merging**